keep-a-changelog.py

```
@@ -0,0 +1,219 @@
+
+Here is the output from `git diff --cached`:
+```
+{}
+```
+"""
+
+@click.command()
+@click.option('--debug', is_flag=True, help='Enable debug logging.')
+@click.option('--model', default='google/gemini-flash-1.5-8b', help='Select AI model.')
+@click.option('--changelog-filename', default='CHANGELOG.md', help='Filename of the changelog.')
+def main(debug, model, changelog_filename):
```

Ruff (0.8.2)

`main` is too complex (11 > 10)

(C901)